src/ui/interaction/files/AvFileUpload/AvFileUpload.vue

```diff
@@ -187,33 +235,58 @@ function onDragLeave () {
 }
 
 function onChange ($event: InputEvent) {
-  emit('update:modelValue', ($event.target as HTMLInputElement)?.value)
-  emit('change', ($event.target as (InputEvent['target'] & { files: FileList }))?.files)
+  const files = ($event.target as HTMLInputElement).files
+  emit('change', files as FileList)
+  emit('update:modelValue', files?.[0] ?? null)
+}
+
+const isPreview = computed(() => props.fileName || (props.modelValue && !props.enableMultiple))
+
+const uploadLabelAttrs = computed(() => {
+  return {
+    'for':
+    props.id,
+    'class':
+    [
+      'fr-upload-group',
+      {
+        'fr-upload-group--error': props.error,
+        'fr-upload-group--valid': props.validMessage,
+        'fr-upload-group--disabled': props.disabled,
+        'drag-over': isDragging.value,
+      },
+    ],
+    'aria-label':
+    props.ariaLabel,
+    'onDragover':
+    onDragOver,
+    'onDragleave':
+    onDragLeave,
+    'onDrop':
+    onDrop,
+  }
+})
```

Comment:
```suggestion
const uploadLabelAttrs = computed(() => {
  return {
    'for': props.id,
    'class': [
      'fr-upload-group',
      {
        'fr-upload-group--error': props.error,
        'fr-upload-group--valid': props.validMessage,
        'fr-upload-group--disabled': props.disabled,
        'drag-over': isDragging.value,
      },
    ],
    'aria-label': props.ariaLabel,
    'onDragover': onDragOver,
    'onDragleave': onDragLeave,
    'onDrop': onDrop,
  }
})
```
Je pense et j'espère que le linter accepte ça